.github/workflows/py-workflow.yml

```diff
@@ -159,16 +169,14 @@ jobs:
           -Dsonar.scm.provider=git
           -Dsonar.qualitygate.wait=true
           -Dsonar.qualitygate.timeout=300
+          -Dsonar.python.coverage.reportPaths=coverage.xml
```

Comment:
what happens if coverage.xml is not present ?

Reply:
SonarCloud shows a warning but I saw no effects on the execution, see [here](https://github.com/MOV-AI/python-delete-me-david/actions/runs/3082597497/jobs/4983692039#step:10:1).
`2022-09-19T14:19:50.7732182Z 14:19:50.772 WARN: No report was found for sonar.python.coverage.reportPaths using pattern coverage.xml`

Reply:
Done